plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/DynamicFilteringJdbcConfig.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.trino.plugin.jdbc;
+
+import io.airlift.configuration.Config;
+import io.airlift.configuration.ConfigDescription;
+import io.airlift.units.Duration;
+import io.airlift.units.MinDuration;
+
+import javax.validation.constraints.NotNull;
+
+import java.util.concurrent.TimeUnit;
+
+public class DynamicFilteringJdbcConfig
+{
+    private boolean enableDynamicFiltering = false;
+    private Duration dynamicFilteringWaitTimeout = new Duration(0, TimeUnit.MINUTES);
+
+    public boolean isEnableDynamicFiltering()
+    {
+        return enableDynamicFiltering;
+    }
+
+    @Config("enable-dynamic-filtering")
```

Comment:
call it `dynamic-filtering.enabled` for consistancy

Reply:
Sure, I followed https://github.com/trinodb/trino/blob/fe608f2723842037ff620d612a706900e79c52c8/core/trino-main/src/main/java/io/trino/execution/DynamicFilterConfig.java#L59 
or is there a naming rule?

Reply:
That `enable-dynamic-filtering` naming schema is kind of legacy. I didn't want to change it since it was already there.
`dynamic-filtering.enabled` is a better schema

Reply:
Understand, will make changes accordingly, thanks for the explanation